blockchain/chain.go

```diff
@@ -1364,6 +1364,20 @@ func (b *BlockChain) BlockLocatorFromHash(hash *chainhash.Hash) BlockLocator {
 	return locator
 }
 
+func (b *BlockChain) IndexLookupNode(hash *chainhash.Hash) *blockNode {
+	b.chainLock.RLock()
+	node := b.index.LookupNode(hash)
+	b.chainLock.RUnlock()
+	return node
+}
+
+func (b *BlockChain) IndexNodeStatus(node *blockNode) blockStatus {
```

Comment:
An exported function should always have a comment for documentation. https://go.dev/doc/comment

Reply:
Okay, will add

Reply:
The comment is wrong. The function is `IndexNodeStatus` but the comment is saying `NodeStatus`.

Reply:
Ohh, I am really sorry, fixed that